mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/feature/OneHotEncoderOp.scala

```diff
@@ -30,18 +30,19 @@ class OneHotEncoderOp extends MleapOp[OneHotEncoder, OneHotEncoderModel] {
 
     override def load(model: Model)
                      (implicit context: BundleContext[MleapContext]): OneHotEncoderModel = {
+      val dropLast = model.value("drop_last").getBoolean
       if (model.getValue("size").nonEmpty) {
         // Old version of 1HE.
+        val handleInvalid = if (dropLast) HandleInvalid.Keep else HandleInvalid.default
         OneHotEncoderModel(categorySizes = Array(model.value("size").getLong.toInt),
-          // assume HandleInvalid.Keep always since old version of 1HE didn't have a way to specify this
-          handleInvalid = HandleInvalid.Keep,
-          dropLast = model.value("drop_last").getBoolean)
+          dropLast = dropLast,
+          handleInvalid = handleInvalid)
```

Comment:
nit: order of arguments is different from the `else` branch. Also you create a `handleInvalid` variable here but inline it in the assignment below

Reply:
Fixed.